src/cross-browser-ui/static/_locales/cs/messages.json

```diff
@@ -0,0 +1,74 @@
+{
+  "extensionName": {
+    "message": "Bergamot",
+    "description": "Name of the extension"
+  },
+  "extensionDescription": {
+    "message": "Strojový překladač cizích jazyků založený na neuronové síti.",
```

Comment:
We (CUNI) think that "Neuronový strojový překlad v prohlížeči" is a more adequate translation of the original "Neural Machine Translation for the browser."

Reply:
Thank you for the proposal. I updated the PR.